### PR TITLE
Rsa security analytics upgrade

### PR DESCRIPTION
**Pull request by maiorrulz**

### CODE REVIEW

**Comment by meirwah** [comment]
@anara123 can you please review this?

**Comment by anara123** [comment]
@meirwah on it

File: Integrations/integration-RsaNetwitnessSecurityAnalytics.yml [hunk 1]
```diff
@@ -1081,57 +1297,60 @@ script:
...
+        case 'netwitness-im-get-components':
             var components = getComponents(sessionId, args);
-            results = { components: components };
+            results = buildRetValGetComponents(components);
             break;
-        case 'nw-get-events':
+        case 'netwitness-im-get-events':
             var events = getEvents(sessionId, args);
-            results = { events: events };
+            results = buildRetValGetEvents(events);
```
Comment (anara123) [inline Integrations/integration-RsaNetwitnessSecurityAnalytics.yml]: When I execute `!netwitness-im-get-events timeRangeType="ALL_DATA" deviceId="6"`
`==` shouldn't be there

image

File: Integrations/integration-RsaNetwitnessSecurityAnalytics.yml [hunk 2]
```diff
@@ -1081,57 +1297,60 @@ script:
...
+        case 'netwitness-im-add-events-to-incident':
+            var md = [];
             var availableAssignees = getAvailableAssignees(sessionId, args);
             var isSuccess = addEventsToIncident(sessionId, args, incidentManagementId, availableAssignees);
-            results = { success: isSuccess };
+            md.push({ success: isSuccess })
+            results = {Type: entryTypes.note, Contents: { success: isSuccess }, ContentsFormat: formats.json, HumanReadable: tableToMarkdown(command, md), ReadableContentsFormat: formats.markdown};
             break;
-        case 'nw-update-incident':
+        case 'netwitness-im-update-incident':
```
Comment (anara123) [inline Integrations/integration-RsaNetwitnessSecurityAnalytics.yml]: image

You should follow the design https://github.com/demisto/server/issues/7102
image

File: Integrations/integration-RsaNetwitnessSecurityAnalytics.yml [hunk 3]
```diff
@@ -1081,57 +1297,60 @@ script:
...
             break;
-        case 'nw-get-alert-details':
+        case 'netwitness-im-get-alert-details':
             var alert = getAlertById(sessionId, args);
-            results = { alert: alert };
+            results = buildRetValGetAlert(alert);
             break;
-        case 'nw-get-alert-original':
+        case 'netwitness-im-get-alert-original':
             var alert = getOriginalAlertById(sessionId, args);
```
Comment (anara123) [inline Integrations/integration-RsaNetwitnessSecurityAnalytics.yml]: I know this is not related to your changes. But please fix this small issue too.
We get this error when such alert not exists
image

File: Integrations/integration-RsaNetwitnessSecurityAnalytics.yml [hunk 4]
```diff
@@ -1081,57 +1297,60 @@ script:
...
-            results = { alert: alert };
+            results = buildRetValGetAlertOrig(alert);
```
Comment (anara123) [inline Integrations/integration-RsaNetwitnessSecurityAnalytics.yml]: The same with `==` happening here
image
image

File: Integrations/integration-RsaNetwitnessSecurityAnalytics.yml [hunk 5]
```diff
@@ -1081,57 +1297,60 @@ script:
...
+        case 'netwitness-im-get-events':
             var events = getEvents(sessionId, args);
-            results = { events: events };
+            results = buildRetValGetEvents(events);
             break;
-        case 'nw-get-event-details':
-            results = getEventDetails(sessionId, args);
+        case 'netwitness-im-get-event-details':
+            event = getEventDetails(sessionId, args);
+            results = buildRetValGetEventDetails(event);
```
Comment (maiorrulz) [inline Integrations/integration-RsaNetwitnessSecurityAnalytics.yml]: Content deleted

File: Integrations/integration-RsaNetwitnessSecurityAnalytics.yml (hunk 4 above)
Comment (maiorrulz) [inline Integrations/integration-RsaNetwitnessSecurityAnalytics.yml]: demo2 local issue

File: Integrations/integration-RsaNetwitnessSecurityAnalytics.yml (hunk 1 above)
Comment (maiorrulz) [inline Integrations/integration-RsaNetwitnessSecurityAnalytics.yml]: demo2 local issue

File: Integrations/integration-RsaNetwitnessSecurityAnalytics.yml (hunk 3 above)
Comment (maiorrulz) [inline Integrations/integration-RsaNetwitnessSecurityAnalytics.yml]: Fixed

File: Integrations/integration-RsaNetwitnessSecurityAnalytics.yml (hunk 2 above)
Comment (maiorrulz) [inline Integrations/integration-RsaNetwitnessSecurityAnalytics.yml]: Fixed

**Pull request merged**